lib/config.js

```diff
@@ -45,6 +45,14 @@ config.authUin = '000000000'; // to specify the user uin on a local server (with
 config.authnCookieMaxAgeMilliseconds = 30 * 24 * 60 * 60 * 1000;
 config.serverType = 'http';
 config.serverPort = '3000';
+config.serverTimeout = 10 * 60 * 1000; // 10 minutes
```

Comment:
It wasn't strictly necessary to make this configurable, but I figure it doesn't hurt!

Reply:
Want to default this to something a bit shorter? Like maybe 90 seconds? Just so it's not so extreme a change from the current 5 second default.

Reply:
Oh sorry, I misread what this was, please ignore this comment!

Reply:
I'm not really sure what the impact of this is going to be in practice, but I agree that it doesn't seem like it will hurt anything.